### PR TITLE
Simplify dev and start commands and env configuration

### PR DESCRIPTION
**Pull request by AugustoL**
- Now no need to be switching between env varibales for kovan and develop.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,17 +2,19 @@
 # React development variables
 REACT_APP_MARKET_API_URL=https://api.coingecko.com/api/v3
 
-REACT_APP_SUBGRAPH_URL_1=
-REACT_APP_SUBGRAPH_URL_42=
-REACT_APP_SUBGRAPH_URL_LOCAL={local-subgraph}
+REACT_APP_SUBGRAPH_URL_1="https://api.thegraph.com/subgraphs/name/balancer-labs/balancer"
```
Comment (tspoff) [inline .env.example:5]: Are these supposed to have balancer subgraph addresses?

File: .env.example [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
+REACT_APP_SUBGRAPH_URL_42="https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-kovan"
 
-# Supported Network ID (e.g. mainnet = 1, rinkeby = 4, kovan = 42)
-REACT_APP_SUPPORTED_NETWORK_ID="66"
-REACT_APP_SUPPORTED_NETWORK_URL="http://localhost:8545"
+REACT_APP_KEY_MNEMONIC=""
+REACT_APP_KEY_INFURA_API_KEY=""
+REACT_APP_ETH_NETWORK="kovan" or "develop"
```
Comment (tspoff) [inline .env.example:10]: Like the name from number -> name

File: .env.example (hunk 1 above)
Comment (AugustoL) [reply .env.example:5]: No idea, I dont knwo for what we used this env vars.

File: .env.example (hunk 2 above)
Comment (AugustoL) [reply .env.example:10]: what?:eyes:

**Pull request merged**